cfgov/jinja2/v1/_includes/eregs/regulations3k-secondary-navigation.html

```diff
@@ -35,22 +35,44 @@
 {% if nav_items is not defined and has_children is not defined %}
     {% set nav_items, has_children = get_secondary_nav_items(request, page) %}
 {% endif %}
-<nav class="o-secondary-navigation
-            {{ '' if has_children else 'o-secondary-navigation__no-children' }}"
+{% from 'organisms/expandable.html' import expandable with context %}
+<nav class="o-regs3k-navigation o-expandable__padded"
      aria-label="Section navigation">
-    {% import 'molecules/nav-link.html' as nav_link %}
-    {% from 'organisms/expandable.html' import expandable with context %}
-    {%- for subpart in nav_items %}
-        {% set sec_nav_settings = {
-            'label': subpart.subpart_heading ~ ' ' ~ subpart.title ~ ' ' ~ subpart.section_range|safe,
-            'hide_cue_label': true,
-            'is_expanded': nav_items[subpart].expanded
-        } %}
-        {% call() expandable(sec_nav_settings) %}
-        {% for section in nav_items[subpart].sections %}
-            {{ nav_link.render(section.title, section.url, true, section.active, true) }}
+    <button class="o-regs3k-navigation_header o-expandable_header o-expandable_target o-expandable_target__collapsed">
+        <span class="h4 o-expandable_header-left o-expandable_label">
```

Comment:
I think you want to change the `h4` for `h5` rather than manage the type changes in the styles.